docs/maui/behaviors/user-stopped-typing-behavior.md

```diff
@@ -0,0 +1,106 @@
+ms.date: 04/13/2022
+---
+
+# UserStoppedTypingBehavior
+
+[!INCLUDE [docs under construction](../includes/preview-note.md)]
+
+The `UserStoppedTypingBehavior` is a `Behavior` that allows the user to trigger an action when a user has stopped data input an `Entry`. Examples of its usage include triggering a search when a user has stopped entering their search query.
+
+## Syntax
+
+### XAML
+
+The `UserStoppedTypingBehavior` can be used as follows in XAML:
+
+```xaml
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:toolkit="http://schemas.microsoft.com/dotnet/2022/maui/toolkit"
+             x:Class="MyLittleApp.MainPage">
+     
+           <Entry Placeholder="Start typing when you stop the behavior will trigger...">
+                <Entry.Behaviors>
+                    <toolkit:UserStoppedTypingBehavior 
+                        Command="{Binding SearchCommand}"
+                        StoppedTypingTimeThreshold="1000"
+                        MinimumLengthThreshold="3"
+                        ShouldDismissKeyboardAutomatically="True" />
+                </Entry.Behaviors>
+            </Entry>
+</ContentPage>
+```
+
+### C#
+
+The `UserStoppedTypingBehavior` can be used as follows in C#:
+
+```csharp
+class UserStoppedTypingBehaviorPage : ContentPage
+{
+    public UserStoppedTypingBehaviorPage()
+    {
+        var behavior = new UserStoppedTypingBehavior()
+        {
+            StoppedTypingTimeThreshold = 1000,
+            MinimumLengthThreshold = 3,
+            ShouldDismissKeyboardAutomatically = true
+        };
+        behavior.SetBinding(UserStoppedTypingBehavior.CommandProperty, nameof(ViewModel. SearchCommand);
+
+        var entry = new Entry
+        {
+            Placeholder = "Start typing when you stop the behavior will trigger..."
+        };
+        entry.Behaviors.Add(behavior);
+    }
+}
+```
+
+### C# Markup
+
+Our [`CommunityToolkit.Maui.Markup`](../markup/markup.md) package provides a much more concise way to use this converter in C#.
+
+```csharp
+using CommunityToolkit.Maui.Markup;
+
+class UserStoppedTypingBehaviorPage : ContentPage
+{
+    public UserStoppedTypingBehaviorPage()
+    {
+        Content = new Entry
+            {
+                Placeholder = "Start typing when you stop the behavior will trigger..."
+            }
+            .Behaviors(new UserStoppedTypingBehavior
+            {
+                StoppedTypingTimeThreshold = 1000,
+                MinimumLengthThreshold = 3,
+                ShouldDismissKeyboardAutomatically = true
+            }.Bind(
+                UserStoppedTypingBehavior.CommandProperty, 
+                nameof(ViewModel. SearchCommand));                   
```

Comment:
```suggestion
        Content = new Entry
        {
            Placeholder = "Start typing when you stop the behavior will trigger..."
        }
        .Behaviors(new UserStoppedTypingBehavior
        {
            StoppedTypingTimeThreshold = 1000,
            MinimumLengthThreshold = 3,
            ShouldDismissKeyboardAutomatically = true
        }
        .Bind(
                UserStoppedTypingBehavior.CommandProperty, 
                nameof(ViewModel. SearchCommand));                   
```

Minor code formatting fixes. Not sure if this is better, but at least more consistent? 😅